skynet/modules/ttt/summaries/processor.py

```diff
@@ -71,8 +71,12 @@ async def process(payload: DocumentPayload, job_type: JobType, model: ChatOpenAI
         chain = load_summarize_chain(current_model, chain_type="map_reduce", combine_prompt=prompt, map_prompt=prompt)
 
     result = await chain.ainvoke(input={"input_documents": docs})
+    formatted_result = result['output_text'].replace('Response:', '', 1).strip()
```

Comment:
Why is this necessary now?

Reply:
I added this in order to minimise randomness in generated responses (e.g.: `sure, here's a summary of...`)